tests/components/binary_sensor/test_command_line.py

```diff
@@ -64,7 +65,8 @@ def test_template(self):
         data = command_line.CommandSensorData('echo 10')
 
         entity = command_line.CommandBinarySensor(
-            self.hass, data, 'test', '1.0', '0', '{{ value | multiply(0.1) }}')
+            self.hass, data, 'test', 'None', '1.0', '0',
```

Comment:
`'None'` is not a valid command class. You're also not testing if it is added to the outputted attributes